app/src/sharedTest/java/org/oppia/android/app/testing/NavigationDrawerActivityTest.kt

```diff
@@ -400,6 +403,31 @@ class NavigationDrawerActivityTest {
     }
   }
 
+  // TODO(#2535): Unable to open NavigationDrawer multiple times on Robolectric
+  @RunOn(TestPlatform.ESPRESSO)
+  @Test
+  fun testNavDrawer_openNavDrawer_dev_switchProfile_cancel_devIsSelected() {
```

Comment:
The mode name is `debug` not `dev` -- ditto everywhere else.

Reply:
Done. PTAL.